src/commonMain/kotlin/fr/acinq/lightning/channel/ChannelException.kt

```diff
@@ -84,7 +84,6 @@ data class CannotSignWithoutChanges                (override val channelId: Byte
 data class CannotSignBeforeRevocation              (override val channelId: ByteVector32) : ChannelException(channelId, "cannot sign until next revocation hash is received")
 data class UnexpectedRevocation                    (override val channelId: ByteVector32) : ChannelException(channelId, "received unexpected RevokeAndAck message")
 data class InvalidRevocation                       (override val channelId: ByteVector32) : ChannelException(channelId, "invalid revocation")
-data class RevocationSyncError                     (override val channelId: ByteVector32) : ChannelException(channelId, "revocation sync error")
```

Comment:
This exception is what made us force close as soon as we detect that our peer is late. We now explicitly handle this case with `SyncResult.Failure.RemoteLate`.